src/interactions/keyInteraction.ts

```diff
@@ -11,10 +11,13 @@ export module Interactions {
      */
     private _positionDispatcher: Plottable.Dispatchers.Mouse;
     private _keyDispatcher: Plottable.Dispatchers.Key;
-    private _keyCodeCallbacks: { [keyCode: string]: Utils.CallbackSet<KeyCallback> } = {};
+    private _keyPressKeyCodeCallbacks: { [keyCode: string]: Utils.CallbackSet<KeyCallback> } = {};
+    private _keyReleaseKeyCodeCallbacks: { [keyCode: string]: Utils.CallbackSet<KeyCallback> } = {};
```

Comment:
While we're modifying this code, I think we can invest in a little effort of code cleanliness.  The way I'm reading the above, a `d3.map()` or a `Utils.Map<string, Utils.CallbackSet<KeyCallback>>` would also work as the type for the above here and it would be in my view more readable and more consitent with the rest of the codebase.